### PR TITLE
Added spider for Allegheny County Jail Oversight Board

### PR DESCRIPTION
**Pull request by woodyeagle**
New Spider

### CODE REVIEW

File: city_scrapers/spiders/jail_oversight.py [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        for possible_meeting in response.css("div.entry-content p"):
+            #        for item in response.css("div.entry-content p").re('.*'):
```
Comment (ben-nathanson) [inline city_scrapers/spiders/jail_oversight.py:23]: Be sure to remove this ghost code before making a final push.

File: city_scrapers/spiders/jail_oversight.py [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+                )
+
+                meeting["status"] = self._get_status(meeting)
+                meeting["id"] = self._get_id(meeting)
+
+                yield meeting
+
+    def _parse_start(self, item):
+        """Parse start datetime as a naive datetime object."""
+        #        datetext = re.search(r"[ADFJMNOS][a-z]{2,8}\s[1-3]?[0-9],\s[12][0-9]{3}", item)
```
Comment (ben-nathanson) [inline city_scrapers/spiders/jail_oversight.py:49]: Be sure to remove this ghost code here as well.

File: city_scrapers/spiders/jail_oversight.py [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+        return datetime.combine(meeting_date, time(16, 0))
+
+    def _parse_end(self, item):
+        """Parse end datetime as a naive datetime object. Added by pipeline if None"""
+        return None
+
+    def _parse_location(self, item):
+        """Parse or generate location."""
+        meeting_date = datetime.strptime(item, "%B %d, %Y")
+        if meeting_date > datetime(2020, 5, 1):
```
Comment (ben-nathanson) [inline city_scrapers/spiders/jail_oversight.py:60]: I like how you dealt with this problem! Ideally, there would be a way to determine the location more dynamically (e.g. regex the event for the words "online") but the structure of the Jail Oversight Board page makes that unlikely.

Comment (ben-nathanson) [reply city_scrapers/spiders/jail_oversight.py:60]: Assuming the Board returns to in-person meetings at some point, we will need to refactor this in the future. But this is a tiny amount of tech debt.

File: tests/test_jail_oversight.py [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+
+freezer = freeze_time("2020-08-07")
+freezer.start()
+
+parsed_items = [item for item in spider.parse(test_response)]
+
+freezer.stop()
+
+
+# def test_tests():
```
Comment (ben-nathanson) [inline tests/test_jail_oversight.py:25]: Good to see you taking advantage of tests! Two general points:

1. Be sure to remove commented-out code throughout this file. 
1. Make sure that you're committing the `jail_oversight.html` file. This is why our tests failed on the most recent commit in this PR. `jail_oversight.html` should have been created when you ran `scrapy genspider ...`. We'll also be available this Friday to go into more depth on this if that's helpful. Join our Meetup [here](https://www.meetup.com/codeforpgh/events/tfsjzrybcnbnc/).

**Review by ben-nathanson — CHANGES_REQUESTED** [review]

**Review by mishugana — APPROVED** [review]

**Pull request merged**